metaworld/envs/mujoco/sawyer_xyz/v2/sawyer_faucet_open_v2.py

```diff
@@ -43,36 +45,32 @@ def model_name(self):
 
     @_assert_task_is_set
     def evaluate_state(self, obs, action):
-        reward, reachDist, pullDist = self.compute_reward(action, obs)
+        (reward, tcp_to_obj, _, target_to_obj, object_grasped,
+         in_place) = self.compute_reward(action, obs)
+
         info = {
-            'reachDist': reachDist,
-            'goalDist': pullDist,
-            'epRew': reward,
-            'pickRew': None,
-            'success': float(pullDist <= 0.05),
+            'success': float(target_to_obj <= 0.11),
```

Comment:
Same here. suspiciously high

Reply:
I computed what the distance of the site was to the goal at the time that it looked like it was a few cm away from the goal. 

That was the final output.

Reply:
Looks like it's 0.07 now, which makes a lot more sense